agent/engine/docker_task_engine_test.go

```diff
@@ -156,6 +156,7 @@ func TestBatchContainerHappyPath(t *testing.T) {
 	default:
 	}
 	eventsReported.Wait()
+	client.EXPECT().DescribeContainer(gomock.Any()).AnyTimes()
```

Comment:
So that sounds like it would be called a maximum of once, not arbitrary times.  Since it gets called later, you might want to set this to `Times(1)` and preserve the later line that sets `AnyTimes()`.